docs/dev/project_management.rst

```diff
@@ -15,6 +15,54 @@ track bugs, enhancements, support requests, and just about any other work that g
 into the project. Try to make sure that issues have informative tags so we can find
 them easily.
 
+
+-------------------------------------------------------------------------------
+Bug triage
+-------------------------------------------------------------------------------
+
+When a new issue is discovered, we need to determine how urgent it is to
+address.
+
+Our core offering is complete, connected, and granular data. Issues that
+interrupt the availability of that data are of highest importance.
+
+However, not all datasets in PUDL are the same; some are mature and have many
+downstream users; others are more experimental. We've split them into "tier 1"
+and "tier 2" groups below.
+
+**Tier 1 datasets**
+* FERC 1 schedules XYZ
+* EIA 860 - XYZ tables
+* EIA 923 - XYZ tables
+* EPA CEMS
+
+**Tier 2 datasets**
+Everything else
+
+This then informs some reliability goals:
+
+For *tier 1* tables:
+
+- latest source data is incorporated into PUDL within 1 month of publication
```

Comment:
This is faster than our actual funding levels allow for at present, we're running on a quarterly integration calendar for our sub-annual datasets for now.